src/Elevator.java

```diff
@@ -1,4 +1,44 @@
+import java.lang.Math;
+import java.time.LocalTime;
+import java.util.ArrayList;
 
-public class Elevator {
+public class Elevator implements Runnable {
+  private int currentFloor;
+  private boolean isStationary;
+  private Middleman middleman;
+  private Direction direction;
+  private ArrayList<ElevatorButton> buttons;
 
+  public Elevator(Middleman middleman) {
+    this.middleman = middleman;
+    isStationary = true;
+    currentFloor = 0;
+    direction = Direction.UP;
+    buttons = new ArrayList<ElevatorButton>();
+
+    //Create an array of elevator buttons
```

Comment:
remove the // comment, I've learned it's not good practice to add inline comments unless the logic is difficult to understand or you need a reminder for something